apps/teamsjs-diagnostic-app/src/apis/index.tsx

```diff
@@ -0,0 +1,11 @@
+export * from './AppApi';
+export * from './AuthenticationStart';
+export * from './AppInstallDialogApi';
```

Comment:
Why isn't `AuthenticationEnd` in this list?

Reply:
Added it in!